scripts/ci/openshift.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# A collection of OpenShift related reusable bash functions for CI
+
+SCRIPTS_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+source "$SCRIPTS_ROOT/scripts/lib.sh"
+
+set -euo pipefail
+
+scale_worker_nodes() {
```

Comment:
Why is it necessary to scale the worker nodes in a QA environment?

Reply:
> Why is it necessary to scale the worker nodes in a QA environment?

This is for postgres i.e. the postgres overhead + compliance needs more CPU.